Editor/CreateBlendTree.cs

```diff
@@ -0,0 +1,32 @@
+using System.IO;
+using UnityEditor;
+using UnityEditor.Animations;
+using UnityEditor.ProjectWindowCallback;
+using UnityEngine;
+
+namespace nadena.dev.modular_avatar.core.editor
+{
+    public class CreateBlendTree : Editor
```

Comment:
Please make this an `internal static class` as this shouldn't be an exposed API.

Reply:
Done.